tests/plugins/asynctest.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+"""This plugin is used to check that async method calls are working correctly.
+
+The plugin registers a method `callme` with an argument. All calls are
+stashed away, and are only resolved on the fifth invocation. All calls
+will then return the argument of the fifth call.
+
+"""
+from lightning import Plugin
+
+plugin = Plugin()
+requests = []
```

Comment:
you can just dynamically attach this to object instance
```
plugin.requests = []
```
Rather that using globals below.

I suspect in the future people may also just subclass Plugin and add their local state there

Reply:
Right, will fixup

Reply:
Decided to initialize with a  `@plugin.init()` decorator to make sure we have the new field whenever it is used.